src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -1097,23 +1070,10 @@ public void Save(Couchbase.Lite.Linq.IDocumentModel model)
         /// <exception cref="InvalidOperationException">Thrown if this method is called after the database is closed</exception>
         [Obsolete("AddChangeListener is deprecated, please use GetDefaultCollection().AddChangeListener")]
         public ListenerToken AddChangeListener([@CanBeNull] TaskScheduler scheduler,
-            [@NotNull] EventHandler<DatabaseChangedEventArgs> handler)
+            [@NotNull] EventHandler<CollectionChangedEventArgs> handler)
```

Comment:
The Database Change Listener will no longer just listen for a DatabaseChange event starts from CBL v3.1 since it should listener for a default collection change event. DatabaseChangedEventArgs will not work with the existing setup anymore. Thoughts?

Reply:
Wrap it so that it still returns databasechangedeventargs.  Document that it will only function for the default collection.